src/composables/mvt-styles/mvt-styles.composable.ts

```diff
@@ -145,46 +147,63 @@ export default function useMvtStyles() {
         med_road_style[i].color = element
       })
       const hillshadeIndex = med_road_style.findIndex(
-        el => el.unlocalized_label === 'Hillshade'
+        el => el.label === 'Hillshade'
       )
       med_road_style[hillshadeIndex].visible = simpleStyle.hillshade
       /*
-        .filter(el => el.unlocalized_label === 'Hillshade')
+        .filter(el => el.label === 'Hillshade')
         .forEach(el => {
           el.visible = simpleStyle.hillshade
         })*/
```

Comment:
I guess this will be needed in the future?

Reply:
no, the hillshade attribute is already present. The style consists of color elements and the hillshade element which have a slightly different behaviour

Reply:
can't we remove the commented code then?

Reply:
sorry, I misread the comment, mea culpa

Reply:
removed